### DISCUSSION TITLE
Issue with Checksum

### DISCUSSION THREAD

**Discussion by afaucogney** [ge-org/multiplatform-swiftpackage#28] [Q&A]
My question is not directly related to this great plugin, but I use it build a framework, the zip and the package.swift.
I put that on a gitlab repo, with the download url pointing to a permalink of the zip file.

When I tried to setup my ios project with this dependency, I get an issue of checksum ! 
When I download the zip file, the checksum is the good one.

Any idea ?

**Comment by ge-org** [comment]
Hi @afaucogney,

thanks for your feedback. The checksum is actually calculated by the `swift` binary. So it should be correct.
How did you integrate the package into the Xcode project?
